Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -2048,6 +2082,20 @@ NonnullRefPtr<YieldExpression> Parser::parse_yield_expression()
     return create_ast_node<YieldExpression>({ m_state.current_token.filename(), rule_start.position(), position() }, move(argument), yield_from);
 }
 
+NonnullRefPtr<AwaitExpression> Parser::parse_await_expression()
+{
+    VERIFY(!m_state.in_formal_parameter_context);
```

Comment:
Why is this a `VERIFY`? It should just give a syntax error in that case right?
Now:
```js
async function f(a = await 1) {}
```
crashes.

Reply:
No clue, must have made some assumptions while writing that, fixed.